src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacClozeValidator.java

```diff
@@ -148,4 +162,47 @@ public final QuestionValidationResponse validateQuestionResponse(final Question
         return new QuestionValidationResponse(question.getId(), answer, responseCorrect, feedback, new Date());
     }
 
+    @Override
```

Comment:
With this all being duplicated from the `IsaacItemQuestionValidator`, would it work to not have this and instead have line 97 as `List<Choice> orderedChoices = new IsaacItemQuestionValidator().getOrderedChoices(clozeQuestion.getChoices());`? I'm not entirely sure this would even work though

Reply:
I was thinking of extending `IsaacItemQuestionValidator`, but since `validateQuestionResponse` is always declared `final` I wasn't able to do it that way. 
I've moved the code in `getOrderedChoices` to a static method in `IsaacItemQuestionValidator`. This lets me refer to it in both places without a problem.